README.md

```diff
@@ -46,6 +46,18 @@ A fixture file is available, you can import its data with :
 $ yarn fixtures
 ```
 
+You can also import more than 2000 contacts in one batch with this other command :
+```
+$ yarn fixtures:massive
+```
+
+### Services
+
+Services can be triggered (and so tested) with `cozy-konnector-dev` by using yarn command. A token will be generated the first time to grant access to `cozy-konnector-dev`, and you may need to delete it (`.token.json`) and regenerate it again after a while.
+```
```

Comment:
Same here

Reply:
I think this line belongs to a general "How to run a service" in the cozy-konnector-libs repo. The docs from contacts could link to the general docs. Otherwise we'd have to repeat this line in each one of our repo. What do you think ?

Reply:
See https://github.com/konnectors/libs/pull/707